src/source/Signaling/FileCache.h

```diff
@@ -0,0 +1,40 @@
+/*******************************************
+Signaling internal include file
+*******************************************/
+#ifndef __KINESIS_VIDEO_WEBRTC_FILE_CACHE__
+#define __KINESIS_VIDEO_WEBRTC_FILE_CACHE__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+#define DEFAULT_CACHE_FILE_PATH                         "./.SignalingCache"
+#define MAX_SIGNALING_CACHE_ENTRY_TIMESTAMP_STR_LEN     10
+/* Max length for a serialized signaling cache entry. 8 accounts for 6 commas and 1 newline
+ * char and null terminator */
+#define MAX_SERIALIZED_SIGNALING_CACHE_ENTRY_LEN        MAX_CHANNEL_NAME_LEN + MAX_ARN_LEN + MAX_REGION_NAME_LEN + \
+                                                        MAX_SIGNALING_ENDPOINT_URI_LEN * 2 + \
+                                                        MAX_SIGNALING_CACHE_ENTRY_TIMESTAMP_STR_LEN + 8
+#define MAX_SIGNALING_CACHE_ENTRY_COUNT                 32
```

Comment:
Oh, I see. We are not using generic MAX_URI_LEN so indeed the entries will be smaller